### PR TITLE
School change-up

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

1st Floor
image

2nd Floor
image

## Why It's Good For The Game

Head Mapper said the school was lacking so.
## Changelog

:cl:
expansion: Expands the School a little bit
/:cl:

### CODE REVIEW

**Comment by Koshenko** [comment]
Overall, really like it.  I'll leave final say up to Original.  I'm not sure of his thoughts on map edits in regards to the terminals, but everything else seems very reasonable.  Just ran the CI suite since it doesn't auto run for new contributors.  Ignore the integration test errors; they're mostly longstanding issues with how the shuttle is hard-coded.

File: mojave/code/datums/documents.dm [hunk 1]
```diff
@@ -102,3 +102,8 @@
 /datum/terminal/document/toystore_notetoself // same as above, immersive
 	title = "NOTE TO SELF"
 	content = "remember to get clay and paint for train diorama"
+
+/datum/terminal/document/principals_office // School Principal's Office - First Log
+	title = "Unruly Brats"
+	content = "Another week another damn prankster. This time it was ,the usual, Steve Brown, he decided it would be a 'funny' prank to clog all the female toilets and wait outside the toilets and laugh as all the students came running out. \
```
Comment (ProfessorPopoff) [inline mojave/code/datums/documents.dm]: > This time it was ,the usual, Steve Brown

Bit of a formatting error here.

**Comment by ProfessorPopoff** [comment]
Hi! Thanks for expressing interest. I greatly appreciate it. The review is pretty brief here for the mapping side of it, I already dropped a small comment on the document.

image
Avoid nudging those trashcans. I'll probably have them moved down in-code, or given directions so they're able to be put in better fitting locations.

image
You've got the opportunity to use random wasteland clothing spawners. You can find them under /obj/effect/spawner/random/ms13/clothing/under

image
While you're in the area- If you could remove/replace these coats with our own variants, That'd be awesome.

image
Avoid using these, as we're going to ideally have a thirst system in place, and they're scuffed with TG mechanics and ungodly amounts of water in them.

That's pretty much all I've got to say about it. Thanks again.

File: mojave/code/datums/documents.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply mojave/code/datums/documents.dm]: I removed the commas, would that suffice?

**Comment by ProfessorPopoff** [comment]
Everything seems fine. I'll merge it once checks 'pass'. Thanks again for your efforts, I appreciate the help.

**Pull request merged**